common/autoware_auto_perception_rviz_plugin/src/object_detection/object_polygon_detail.cpp

```diff
@@ -212,7 +212,7 @@ visualization_msgs::msg::Marker::SharedPtr get_label_marker_ptr(
 visualization_msgs::msg::Marker::SharedPtr get_shape_marker_ptr(
   const autoware_auto_perception_msgs::msg::Shape & shape_msg,
   const geometry_msgs::msg::Point & centroid, const geometry_msgs::msg::Quaternion & orientation,
-  const std_msgs::msg::ColorRGBA & color_rgba)
+  const std_msgs::msg::ColorRGBA & color_rgba, const double line_width)
```

Comment:
```suggestion
  const std_msgs::msg::ColorRGBA & color_rgba, const double & line_width)
```

reference value is better?